src/tax/capitalGains.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import moment from "moment";
+import Big from "big.js";
+
+import type {BasisFrame} from "./basis";
+import {LIFOCostBasisCalculator} from "./basis";
+import type {Transaction} from "../core/transaction";
+import {transactionTypes} from "../core/transaction";
+
+export type CapitalGainsRecord = {
+  amount: Big,
+  ticker: string,
+  acquiredDate: moment,
+  disposedDate: moment,
+  unitCost: Big, // per-coin (not aggregate)
```

Comment:
I don’t get it. What’s the difference between `unitCost` and `price` if
they are both per-coin (not aggregate)? I thought that a transaction’s
`price` was in USD, and `amount` was in the token given by `ticker`, so
that `price = amount * currentExchangeRate`. But then `price` _is_ an
aggregate, right?

Some docs here and on Transaction will probably help.

Reply:
renamed price to unitProceeds, and added a comment. 
so, the $ value of gains to pay taxes on is `amount * (unitProceeds - unitCost)`